app/controllers/partner_groups_controller.rb

```diff
@@ -0,0 +1,46 @@
+class PartnerGroupsController < ApplicationController
+  def index
+    @partner_groups = current_organization.partner_groups
+
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def new
+    @partner_group = current_organization.partner_groups.new
+    @item_categories = current_organization.item_categories
+  end
+
+  def create
+    @partner_group = current_organization.partner_groups.new(partner_group_params)
+    if @partner_group.save
+      # Redirect to groups tab in Partner page.
+      redirect_to partners_path + "#nav-partner-groups", notice: "Partner group added!"
```

Comment:
I wonder if there's a better way to do this than string concatenating the `#` anchor, but we can save that for a future improvement

Reply:
Yeh I wasn't too sure, how to achieve that without making another route.